Source/Core/DolphinQt2/FIFOWindow.h

```diff
@@ -0,0 +1,63 @@
+// Copyright 2017 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#pragma once
+
+#include <QDialog>
+
+class QCheckBox;
+class QDialogButtonBox;
+class QLabel;
+class QPushButton;
+class QSpinBox;
+
+class FIFOWindow : public QDialog
```

Comment:
`FIFOWindow` isn't descriptive, it should be something like `FIFOPlayerWindow` (to match DolphinWX) or something that tells a bit more about what it's doing (loads FIFOs, records FIFOs, shows information on FIFOs, allows configuring FIFOPlayer, etc.).